taskana-adapter-taskana-connector/src/main/java/pro/taskana/adapter/taskanaconnector/api/impl/TaskInformationMapper.java

```diff
@@ -96,8 +98,10 @@ public Task convertToTaskanaTask(ReferencedTask referencedTask)
     private void setTimestampsInTaskanaTask(TaskImpl taskanaTask, ReferencedTask camundaTask) {
         Instant created = convertStringToInstant(camundaTask.getCreated(), Instant.now());
         taskanaTask.setCreated(created);
-        Instant due = convertStringToInstant(camundaTask.getDue(), created.plus(Duration.ofDays(3)));
-        taskanaTask.setDue(due);
+        if (created != null) {
+            Instant due = convertStringToInstant(camundaTask.getDue(), created.plus(Duration.ofDays(3)));
```

Comment:
Have a look at this: https://taskana.atlassian.net/wiki/spaces/TAS/pages/881164289/Camunda+BPM+Connector

'due' should be mapped to 'due'. There should be no need to to calculate anything. This should happen in TASKANA core based on the classification.